src/selectors/url/report/getPlayerName.js

```diff
@@ -2,5 +2,8 @@ import getMatch from './getMatch';
 
 export default state => {
   const match = getMatch(state);
-  return match ? match.params.playerName : null;
+  if(match && match.params.player){
+    return match.params.player.indexOf('-') > -1 ? match.params.player.split('-')[1] : match.params.player;
```

Comment:
Split might break the result in more than 2 parts, which would break this. How about:

```jsx
const index = match.params.player.indexOf('-');
return index !== -1 ? match.params.player.substr(index) : match.params.player;
```

Reply:
That's a much better solution than what I had, got it updated!